CodeGenerator/Sources/CodeGenerator/patch.swift

```diff
@@ -12,10 +12,6 @@ import SwiftyJSON
 // Patch operations
 //
 let servicePatches : [String: [Patch]] = [
-    "Backup" : [
-        // Backup clashes with a macOS framework, so need to rename the framework
-        Patch(.replace, entry:["serviceName"], value:"AWSBackup", originalValue:"Backup")
```

Comment:
This precursor patch (and the one below) likely points out we should've done this all along. Thank you!